resources/js/beatmap-discussions/editor.tsx

```diff
@@ -230,7 +230,12 @@ export default class Editor extends React.Component<Props, State> {
 
   onKeyDown = (event: React.KeyboardEvent<HTMLDivElement>) => {
     const nativeEvent = event.nativeEvent;
-    if (isHotkey('mod+b', nativeEvent)) {
+    if (nativeEvent.key === 'ArrowDown') {
+      // TODO: only move to editable lines?
+      Transforms.move(this.slateEditor, { unit: 'line' });
+    } else if (nativeEvent.key === 'ArrowUp') {
+      Transforms.move(this.slateEditor, { distance: -1, unit: 'line' });
```

Comment:
This was supposed to force the cursor to only move between lines instead of going left/right and getting stuck on parts of different blocks but I guess something broke it since I last wrote and tested it 😑